ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,619 @@
+making it easier to apply common configuration to a group of devices.
+
+The ***all*** line at the top is a default group that contains all ***hosts*** i.e. all switches. Don’t worry too much about that for now.
+
+The ***children:*** keyword is used to define “groups of groups” i.e. just an internal keyword to differentiate between hosts and groups.
+
+The multi-colored figure below is just a visualization of the same text file, showing the different grouping and parent/child relationships:
+
+![Figure: Ansible Inventory](../_media/getting-started/Inventory.png)
+
+The key takeaway is that configuration defined at a higher (parent) level will be inherited by the children, but if needed, this can be
+overridden by specifying it again, i.e. most specific wins.
+
+More information about Ansible inventories can be found [here](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#).
+
+### Organizing host and group variables
+
+AVD makes use of variables defined in the inventory to generate the EOS configuration files necessary to configure the devices. While these
+variables can be stored in the main inventory file described earlier, it is highly recommended to use separate files for host and group
+variables. Like the inventory.yml file, the host and group variables are also stored in YAML files in dedicated folders named accordingly.
+
+### Group Variables
+
+Group variables are defined in YAML files inside the `group_vars` folder:
+
+![Figure: Ansible Group Variables](../_media/getting-started/Group-vars-folder-structure.png)
+
+Please note that the example above is taken from the [ansible-avd-cloudvision-demo](https://github.com/arista-netdevops-community/ansible-avd-cloudvision-demo) repository.
+
+Each file in the `group_vars` folder controls the variables for one of the groups defined in the `inventory.yml` file,
+so settings for the whole network can be specified in `DC1.yml`, and later overridden in `DC1_SPINES.yml` for just the hosts
+inside that group (DC1 spine switches in this case).
+
+A subset of DC1.yml is shown below:
+
+```yaml
+# local users
+local_users:
+  admin:
+    privilege: 15
+    role: network-admin
+    sha512_password: "<password-hash>"
+
+  cvpadmin:
+    privilege: 15
+    role: network-admin
+    sha512_password: "<password-hash>"
+
+  ansible:
+    privilege: 15
+    role: network-admin
+    sha512_password: "<password-hash>"
+
+# Cloud Vision server information
+cvp_instance_ip: <CVP IP address>
+cvp_ingestauth_key: '<key>'
+
+# OOB Management network default gateway.
+mgmt_gateway: <IP of default gateway>
+
+# dns servers.
+name_servers:
+  - <IP of name server #1>
+  - <IP of name server #2>
+
+# NTP Servers IP or DNS name, first NTP server will be preferred, and sourced from Management VRF
+ntp_servers:
+  - 0.<local country>.pool.ntp.org
+  - 1.<local country>.pool.ntp.org
+```
+
+Below is a subset of the DC1_FABRIC file:
+
+```yaml
+# Leaf switch groups
```

Comment:
In this snippet we have 3 different names for the same thing. Leaf switch groups, leaf group and node_group. The correct notion is `node_group`.
It is not true that maximum of two nodes can form a group. But it is correct that if there are exactly two nodes in a node group, they will form _an_ mlag pair.